tests/bgp/test_bgp_as_path_prepend.py

```diff
@@ -0,0 +1,71 @@
+# Helper Functions
+import pytest
+import json
+from tests.common.helpers.assertions import pytest_assert
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t1')
+]
+
+as_path = '54321'
+
+def asn_num(duthost):
+    bgp_summary = json.loads(duthost.shell('vtysh -c "show bgp summary json"')['stdout'])
+    bgp_num = bgp_summary['ipv4Unicast']['as']
+    return bgp_num
+
+def ipadd(duthost):
+    bgp_summary = json.loads(duthost.shell('vtysh -c "show bgp summary json"')['stdout'])
+    peer_key=bgp_summary['ipv4Unicast']['peers'].keys()
+    peer_list = list(peer_key)
+    ip_value = peer_list[-1]
+    return ip_value
+
+# Test Functions
+def test_show_aspath_pre(duthosts, enum_dut_hostname):
```

Comment:
Sounds to me this is one function test, to test if the route-map as-path can be added successfully then restore, should we unit all these 5 testcases to one? Two reasons.
1. If some cases failed in middle, should we still run others? Like if test_show_aspath_post failed, there is no aspath appended, how we can make sure if test_aspath_no_config works?
2. There are pre/after test checks with pytest, it costs extra time, the benefit added by split to 5 testcases doesn't pay the price. 

Maybe you can refer to other testcase in some module, tests/bgp/test_traffic_shift.py.